webpack/published-components.js

```diff
@@ -93,7 +93,8 @@ const publishedComponents = {
   useResizeObserver: "hooks/useResizeObserver.js",
   useAfterFirstRender: "hooks/useAfterFirstRender/index.js",
   useTimeout: "hooks/useTimeout/index.js",
-  usePrevious: "hooks/usePrevious",
+  usePrevious: "hooks/usePrevious.js",
```

Comment:
this will not work because hooks/usePrevious is a library, please change to hooks/usePrevious or hooks/usePrevious/usePrevious.js